python/pyspark/sql/functions.py

```diff
@@ -2155,6 +2155,8 @@ def udf(f=None, returnType=StringType()):
         in boolean expressions and it ends up with being executed all internally. If the functions
         can fail on special rows, the workaround is to incorporate the condition into the functions.
 
+    .. note:: The user-defined functions do not take keyword arguments.
```

Comment:
Let's add `on the calling side` (I meant not on function declaration side) at the end for clarification.